src/RollbarLogHandler.php

```diff
@@ -73,55 +74,11 @@ public function log($level, $message, array $context = [])
             return;
         }
 
-        $context = $this->addContext($context);
```

Comment:
Ahh, but I see how it could work because @ArturMoczulski https://github.com/jenssegers/laravel-rollbar/compare/master...rollbar:master#diff-e31215fa03abb6aa2a7a094e2286af78L112 knows how to do it.